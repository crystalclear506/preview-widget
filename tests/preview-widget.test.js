import { expect, test } from '@jest/globals';
import React from 'react';
import renderer from 'react-test-renderer';
import PreviewWidget from '../src/Components/preview-widget';

jest.useFakeTimers();

test('Component should be rendered correctly', () => {
    const testRenderer = renderer.create(
        <PreviewWidget/>
    );
    
    expect(testRenderer).toMatchSnapshot();
});