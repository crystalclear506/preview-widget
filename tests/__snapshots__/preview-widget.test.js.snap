// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should be rendered correctly 1`] = `
<div
  className="previewWidgetContainer"
>
  <div
    className="previewContainer"
  >
    <img
      alt="No Image"
      src=""
    />
  </div>
  <div
    className="grid"
  >
    <div
      className="thumbnail"
    >
       
      <img
        alt="No Image"
      />
       
    </div>
  </div>
</div>
`;
